bindings/go/utils.go

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2021, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+package ucp
+
+// #include <stdlib.h>
```

Comment:
remove?

Reply:
It needs for cgo to use `free` function.

Reply:
isn't it commented out?

Reply:
https://golang.org/cmd/cgo/#hdr-Using_cgo_with_the_go_command - it's not a comment.
It's cgo preprocessor directive to tell which C headers to include. Everything that goes before `import "C"` comments are interprets cgo as C source:
```
// #include <stdio.h>
// #include <errno.h>
import "C"
```